src/main/java/core/basesyntax/transaction/SetBalanceTransaction.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.transaction;
+
+import core.basesyntax.model.Fruit;
+
+public class SetBalanceTransaction implements TransactionHandler {
```

Comment:
follow name convention
```suggestion
public class BalanceTransaction implements TransactionHandler {
```